azure-templates/powershell-scripts/check-count-size.ps1

```diff
@@ -0,0 +1,79 @@
+param(
+  [string]$archiveDir
+)
+Write-Output "Finding latest successful build of `"main`" for comparison..."
+$allBuildsOfMain = Get-ChildItem `
+  -Path "$archiveDir\NI\export\main\*" `
+  | Sort-Object {$_.Name} -Descending
+ForEach ($build in $allBuildsOfMain)
+{
+  Write-Output "Checking $build..."
+  If (Test-Path -Path "$build")
+  {
+    Write-Output "Checking for .finished file..."
+    If (Test-Path -Path "$build\.finished")
+    {
+      Write-Output "latest successful build: $build"
+      $mainPath = $build
```

Comment:
What happens if this is the first build?

Reply:
If there isn't a successful build of main, it does look like it fails. I re-ran the SLSC 12201 build after renaming the main folder in nirvana:
https://dev.azure.com/ni/DevCentral/_build/results?buildId=4820920&view=logs&j=77e990a9-fa48-5f00-e29b-d41b37ea0dd1&t=005d0013-664e-5c78-ba34-7a6f15ff7a3e

Reply:
Good question - currently it will report that a file could not be found, but I will add in a check to skip on first build.

Reply:
![image](https://user-images.githubusercontent.com/42351034/235733422-cad5cc9a-6fa5-4896-83f1-673e0aee8e36.png)
Instead of failing, now it passes

https://dev.azure.com/ni/DevCentral/_build/results?buildId=4821134&view=logs&j=77e990a9-fa48-5f00-e29b-d41b37ea0dd1&t=005d0013-664e-5c78-ba34-7a6f15ff7a3e

Reply:
![image](https://user-images.githubusercontent.com/42351034/235736115-8dff8988-9f18-4762-a3a6-7f89f13ca695.png)
Same if there's a main but no successful builds:
https://dev.azure.com/ni/DevCentral/_build/results?buildId=4821184&view=logs&j=77e990a9-fa48-5f00-e29b-d41b37ea0dd1&t=005d0013-664e-5c78-ba34-7a6f15ff7a3e